lxd/main_init.go

```diff
@@ -573,15 +583,62 @@ func (cmd *CmdInit) initConfig(client lxd.ContainerServer, config map[string]int
 	return client.UpdateServer(server.Writable(), etag)
 }
 
+// Create the given pools if they don't exist yet.
+func (cmd *CmdInit) initPools(client lxd.ContainerServer, pools []api.StoragePoolsPost) error {
+	for _, pool := range pools {
+		_, _, err := client.GetStoragePool(pool.Name)
+		if err == nil {
+			err = client.UpdateStoragePool(pool.Name, api.StoragePoolPut{
```

Comment:
Yeah, I think for the most part this API call will fail, but I guess there are small things like the new description field that one can update without the API call failing. Anyway, this code ensures that the call is tried, it will be up the the API implementation to decide if it can succeed or not (and that's effectively the real meaning of the mention in the commit message).